### PR TITLE
Introducing hover behaviour and onclick for multicount component

### PR DESCRIPTION
**Pull request by nulikartik**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes
Hover behavior for Multicount component completed, introduced on-click listener on each row and corresponding cursor.
Needs annotation text no-wrap CSS fix and change of icon to SVG.
image

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by micahgodbolt** [comment]
please run `npm run change` and add the changefile

File: packages/dashboard/src/components/MultiCount/MultiCount.tsx [hunk 1]
```diff
@@ -55,24 +64,87 @@ export class MultiCount extends React.Component<IMultiCountProps, {}> {
...
+      AnnotationType[data.type] === 'neutral' ? '' : AnnotationType[data.type] === 'positive' ? 'FlickDown' : 'FlickUp';
+    const getClassNames = classNamesFunction<IMultiCountProps, IMultiCountStyles>();
+    const classNames = getClassNames(
+      getStyles({
+        color: data.color,
+        iconName: changeIconIndicator
+      })
+    );
+    return (
+      <div id="content" className={classNames.hoverCardRoot}>
```
Comment (Raghurk) [inline packages/dashboard/src/components/MultiCount/MultiCount.tsx]: >content [](start = 15, length = 7)

Is there a reason why we have an id?

File: packages/dashboard/src/components/MultiCount/MultiCount.tsx [hunk 2]
```diff
@@ -55,24 +64,87 @@ export class MultiCount extends React.Component<IMultiCountProps, {}> {
...
+    }
+  };
+
+  private _redirectToUrl(href: string | undefined): void {
+    href ? (window.location.href = href) : '';
+  }
+
+  private _onRenderCompactCard = (data: IMultiCountRow): JSX.Element => {
+    const changeIconIndicator =
+      AnnotationType[data.type] === 'neutral' ? '' : AnnotationType[data.type] === 'positive' ? 'FlickDown' : 'FlickUp';
```
Comment (Raghurk) [inline packages/dashboard/src/components/MultiCount/MultiCount.tsx]: >positive [](start = 84, length = 8)

are you sure these will be lower case? Do you want to compare this with tolower?

Comment (Raghurk) [inline packages/dashboard/src/components/MultiCount/MultiCount.tsx]: >FlickDown [](start = 97, length = 9)

Similarly these strings, make sure you define them as constats

**Pull request merged**